src/pages/workspaces/workspace/tools/WorkflowView.js

```diff
@@ -156,8 +176,9 @@ class WorkflowView extends Component {
           [
             div({ style: { flexGrow: 1 } }),
             saving && miniMessage('Saving...'),
-            !untouched && !saving && !modified && miniMessage('Saved!'),
-            modified && buttonPrimary({ disabled: saving, onClick: () => this.save() }, 'Save')
+            saved && !saving && !modified && miniMessage('Saved!'),
+            modified && buttonPrimary({ disabled: saving, onClick: () => this.save() }, 'Save'),
```

Comment:
I was worried this could be jarring, but it isn't. Is it possible to scroll these buttons out of view? If so, it might be better to keep them here (just disabled) so I know where they are when I make modifications to a long form.

Reply:
We need to make the scrolly area take up the remaining space on the page. Then the top half wouldn't move unless the user has a really scrunched-up browser window.